subsys/bluetooth/controller/ll_sw/ull_iso_internal.h

```diff
@@ -10,3 +10,4 @@ void ull_iso_datapath_release(struct ll_iso_datapath *dp);
 void ll_iso_rx_put(memq_link_t *link, void *rx);
 void *ll_iso_rx_get(void);
 void ll_iso_rx_dequeue(void);
+void ll_iso_transmit_test_send_sdu(uint16_t handle, uint32_t ticks_at_expire);
```

Comment:
A newline is missing at the end of the file?

Reply:
I was not going to request re-review until it passed CI